static/js/lib/validation/profile.js

```diff
@@ -299,5 +299,10 @@ export const validateFinancialAid = (edit: FinancialAidState): FinancialAidValid
   if (!edit.checkBox) {
     errors['checkBox'] = 'You must agree to these terms';
   }
+
+  if (edit.income && !/^\d+$/.test(edit.income)) {
```

Comment:
`!/^\d+$/.test(edit.income)` and `/\D/.test(edit.income)` are, as far as I can tell, totally equivalent - I'm just advocating for the latter because the regular expression is simpler.